### DISCUSSION TITLE
Viewing custom error page

### DISCUSSION THREAD

**Discussion by jensmeindertsma** [vercel/next.js#16510] [Help]
How can I trigger my custom error page to display so I can verify the design of it?

In development, it is not shown, because a stack trace dialog is shown instead. 

But if I want to style my custom error page, I need to verify the design. How can I trigger this page to show up in development / production mode?

**Comment by Belco90** [comment]
Hey there, just run into this question. From Next.js docs:

> `pages/_error.js` is only used in production. In development you’ll get an error with the call stack to know where the error originated from.

So you need to build your app in production mode to see the error with `npm run build && npm run serve` (or equivalent with yarn). Remember every time you change your code in this mode you need to stop and rerun the previous commands.
